src/ui/pages/BridgeForm/LabeledNetworkSelect/LabeledNetworkSelect.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useMemo } from 'react';
+import cx from 'classnames';
+import type { NetworkConfig } from 'src/modules/networks/NetworkConfig';
+import { HStack } from 'src/ui/ui-kit/HStack';
+import { NetworkIcon } from 'src/ui/components/NetworkIcon';
+import { VStack } from 'src/ui/ui-kit/VStack';
+import { UIText } from 'src/ui/ui-kit/UIText';
+import type { Networks } from 'src/modules/networks/Networks';
+import { createChain } from 'src/modules/networks/Chain';
+import { Spacer } from 'src/ui/ui-kit/Spacer';
+import DownIcon from 'jsx:src/ui/assets/chevron-down.svg';
+import { UnstyledButton } from 'src/ui/ui-kit/UnstyledButton';
+import { NetworkSelect } from '../../Networks/NetworkSelect';
+import * as styles from './styles.module.css';
+
+function NetworksDisclosureButton({
+  title,
+  value,
+  networks,
+  openDialog,
+}: {
+  title: string;
+  value: string;
+  networks: Networks | null;
+  openDialog: () => void;
```

Comment:
onClick or onDialogOpen?

Reply:
we have `openDialog` everywhere else (I mean in all variants of the `DisclosureButton`), so I decided to keep it as is